src/app/clients/add/submitFormHelpers.ts

```diff
@@ -100,9 +94,8 @@ const updateChildren = async (children: Person[]): Promise<void> => {
             .eq("primary_key", child.primaryKey);
 
         if (errorExists(error, status)) {
-            // TODO VFB-22 Standardize Error Handling
-            throw Error(
-                `Error occurred whilst updating children in the Families table. HTTP Code: ${status}, PostgreSQL Code: ${error?.code}. `
+            throw new Error(
+                "We could not process the request at this time. Please try again later."
```

Comment:
This statement is used multiple times across the same file. Would it be nice to make it a const like `genericErrorMessage` or something alike?

Reply:
I agree, will do!